src/libLookup/Lookup.cpp

```diff
@@ -117,19 +119,41 @@ void Lookup::SendMessageToLookupNodes(
                                                 << ":"
                                                 << node.m_listenPortHost);
 
-        AllLookupNodes.push_back(node);
+        allLookupNodes.push_back(node);
     }
 
-    P2PComm::GetInstance().SendBroadcastMessage(AllLookupNodes, message);
+    P2PComm::GetInstance().SendBroadcastMessage(allLookupNodes, message);
 }
 
-void Lookup::SendMessageToRandomLookupNodeFromBack(
+void Lookup::SendMessageToLookupNodesSerial(
     const std::vector<unsigned char>& message) const
 {
     LOG_MARKER();
 
-    int index = rand() % (NUM_LOOKUP_USE_FOR_SYNC) + m_lookupNodes.size()
-        - NUM_LOOKUP_USE_FOR_SYNC;
+    // LOG_GENERAL("i am here " << to_string(m_mediator.m_currentEpochNum).c_str())
+    vector<Peer> allLookupNodes;
+
+    for (auto node : m_lookupNodes)
+    {
+        LOG_EPOCH(INFO, to_string(m_mediator.m_currentEpochNum).c_str(),
+                  "Sending msg to lookup node " << node.GetPrintableIPAddress()
+                                                << ":"
+                                                << node.m_listenPortHost);
+
+        allLookupNodes.push_back(node);
+    }
+
+    P2PComm::GetInstance().SendMessage(allLookupNodes, message);
```

Comment:
Can we use   m_lookup->SendMessageToLookupNodes(dsblock_message);

Reply:
That will cause the flooding broadcast boom as we met before

Reply:
ok